src/lib.rs

```diff
@@ -350,3 +335,58 @@ impl<'a> Chip8<'a> {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // Screen specific tests
+    #[test]
+    fn test_clear_screen_works() {
+        let mut screen = Screen::new();
+        screen.bitmap[1][2] = true;
+        screen.clear_screen();
+        assert!(!screen.bitmap[1][2]);
+    }
+
+    // Chip8 specific tests
+    #[test]
+    fn test_load_rom() {
+        // TODO: Remove this message once the below comment is verified
+        // Concerned about endianess and how we are currently reading the ROMs.
+        // I *believe* that we are doing it correctly here, but this also
+        // illustrates the annoyance of writing our own mini test ROMs as u8s.
+        // If this is correct, a helper function taking a Vec<u16> and converting
+        // it to a Vec<u8> would simplify the unit testing.
+        // Supposedly chip-8 programs are stored big endian... so the following
```

Comment:
This is true. Please refer to the RCA COSMAC VIP CDP18S711
Instruction Manual page 10 under Memory Write.

Reply:
I'll let you merge this in after you read this.